pkg/kdump/kdump-view.jsx

```diff
@@ -325,6 +385,33 @@ export class KdumpPage extends React.Component {
                                          handleSave={this.props.onSaveSettings} />);
     }
 
+    handleAutomationClick() {
+        const Dialogs = this.context;
+        let enableCrashKernel = '';
+        let kdumpconf = this.props.exportConfig(this.props.kdumpStatus.config);
+        kdumpconf = kdumpconf.replaceAll('$', '\\$');
+        if (this.state.os_release.NAME.includes('Fedora')) {
```

Comment:
`NAME?.includes` please -- there *shouldn't* be os-release files without it, but that's not under our control.